daml-assistant/integration-tests/src/Main.hs

```diff
@@ -27,31 +29,39 @@ import DA.Bazel.Runfiles
 import DamlHelper
 import SdkVersion
 
+-- | Since we run in bash and not in cmd.exe "daml" won’t look for "daml.cmd"
+-- so we use "daml.cmd" directly.
+damlName :: String
+damlName
```

Comment:
Why not just move to using the Shell form of process? In truth, that's what most users will be doing, so its the most accurate test and no need to have daml.cmd vs daml.

Reply:
Ah, I found adjustCP below...

Reply:
Moved it after `adjustCP` and pointed to the docs of `adjustCP`.